relay/messages/message.go

```diff
@@ -55,37 +59,53 @@ type HelloResponse struct {
 	InstanceAddress string
 }
 
-// DetermineClientMsgType determines the message type from the first byte of the message
-func DetermineClientMsgType(msg []byte) (MsgType, error) {
+// ValidateVersion checks if the given version is supported by the protocol
+func ValidateVersion(msg []byte) (int, error) {
```

Comment:
If I look right we never use the returned int.

Reply:
Correct, but might use it if we need to branch based on version